lib/scoreboard_renderer.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class ScoreboardRenderer < ParticipantsController
+  module ScoreboardRendererHelper
+    def javascript_include_tag(*sources)
+      options = sources.extract_options!.merge(debug: false)
```

Comment:
Why?

Reply:
Oh gosh, okay, you want the assets to _always_, even in development, be precompiled into application.whatever.

Reply:
Well, the point is that they _are_ compiled as such, and I want the HTML to point to files that exist.